src/docker-compose-dev.yml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+services:
+
+  db:
+    extends:
+      file: docker-compose.yml
+      service: db
+
+  server:
+    build: ./server
+    image: leaf_api
+    container_name: leaf_api_demo
+    ports:
+      - 5001:5001
+    volumes:
+      - ./server/keys:/.keys
+      - ./server:/app
+      - server-log:/logs
+    environment:
+      - LEAF_APP_DB=${LEAF_APP_DB}
+      - LEAF_CLIN_DB=${LEAF_CLIN_DB}
+      - LEAF_JWT_KEY_PW=${LEAF_JWT_KEY_PW}
+      - UnsecuredIsAdmin=${UnsecuredIsAdminDev}
+    networks:
+      - leaf
+    depends_on:
+      - db
```

Comment:
Do we need this all with `extends`?

Reply:
"services with 'depends_on' cannot be extended"

Reply:
Yes, they will need to be in a separate "common services" file.